### ISSUE TITLE
Gollobin file too big for github

### ISSUE THREAD

**Issue by DavidMc1948** [ScientificPublishing/SciPub#10]
I've created a pdf from Craig's scans which is significantly more legible than the library genesis version. However, it is 250 megs and too big for github. I tried online compression but ended with some sections being low quality. I don't want to spend any more time on that at the moment. A short term fix is to break the book up into a number of sections.

**Comment by artydont** [comment]
Congratulations! Looking forward to resuming reading it TTS.

I created a repo for "help" with a CC0 "No Rights Reserved" license. Will add a CONTRIBUTING.md later but everybody should subscribe to "watch" all Issues in that repo now. Oddly I cannot see an Issues tab here:

https://github.com/ScientificPublishing/help

Will be used to resolve problems like exchanging large or confidential files, configuring stuff needed to do so etc. But aiming for final clear instructions.

Proposed immediate short term solution for this large file below.

## tl;dr

1. Windows users should install Chocolatey now. (MacOS users will use HomeBrew with "brew --cask").
2. All should  install  Brave browser now.
3. I may report back that people should use Brave browser (or latest version of cURL) with links I will email,  if I don't report back that all should install retroshare (with configuration details that I will provide).
4. It would be good to also have a much smaller word processor file from extracting the OCR from the .pdf.
(But we would still need to pass around the large .pdf)
5. Don't forget to "watch" and subscribe to the "help" repo.
## Details

Should simply provide a shared folder on each of our PC's which is not (easily) accessible to others but "just works" between us. Delays caused by not being online at the same time don't matter much.

I would rather avoid resorting to Drop Box or Google Drive short term as that will end up getting repeated use and waste people's time extricating from it later.

I am about to install:

http://retroshare.cc/

If it is too difficult to configure quickly then I would recommend snailmail posting a USB to Craig while I work something out for medium term use.

I am hopeful it will be easy enough to get the "File Sharing" working immediately and add dynamic DNS names instead of having to use IP addresses and a "family" crypto key shortly. I would prefer to start by doing that as we can then add anonymity with i2pd medium term. We will certainly need something at least equally hard to configure long term.
Version 0.6.6 is available for all 3 platforms (Windows, MacOS and Linux) although it has not been updated for 2 years.

Don't waste time splitting files etc as it will require a tutorial on reassembling.

Do consider extracting the OCR text to a word processor file which should be **MUCH** shorter.

Will add a note recommending that @DavidMc1948 and @Ted1307 should install it, with configuration details here if it looks promising (and will email any necessary keys and my IP address). May try to get it working between me and @PetrogradXXII first.

The MacOS Homebrew package for retroshare is here:

https://formulae.brew.sh/cask/retroshare

(Use that to get latest version and any future updates. Ignore the v 0.6.5 for MacOS at the retroshare.cc download page.)

The Windows version at retroshare download page is current. But it is probably desirable to install Chocolately package manager to get updates of that and any other Windows software.

@Ted1307  already has Homebrew installed. Any MacOS user should use this with [--cask](https://github.com/Homebrew/homebrew-cask/blob/master/USAGE.md)

I think Windows users should install [Cholcolatey](https://community.chocolatey.org/) now, even if we don't use retroshare. There are other [Windows package managers](https://en.wikipedia.org/wiki/List_of_software_package_management_systems#Windows) for free software. I don't know whether they are better but I do know [retroshare is available via Chocolatey](https://community.chocolatey.org/packages/retroshare#discussion)

Another that has retroshare (v0.6.6)  is:

[Scoop](https://scoop.sh/#/apps?q=retroshare).

The installation of Chocolatey looks  somewhat intimidating, but should result in simpler access to free software for Windows:

https://chocolatey.org/install

A simplified installation/tutorial course on installation is here (with options for installation via other systems):

https://community.chocolatey.org/courses/installation/installing?method=installing-chocolatey

I'm not recommending immediately installing retroshare until subsequent comment after checking.

But all should take a look at the retroshare docs to see why I want to use something like that instead of a simpler and faster Drop Box solution. But don't get scared off by the details:

https://retrosharedocs.readthedocs.io/en/latest/

Skip all the unpleasant installation/configuration details (my problem, not yours).

The interesting part is just "File Sharing":

https://retrosharedocs.readthedocs.io/en/latest/user-guide/interface/#file-sharing

Again don't worry about the details of that. The concept is a shared folder that "just works" (I hope).

Another option for package management on Windows is to use:

https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux

Then we could use HomeBrew on all 3 platforms. I think that would only be complicating things now but may become relevant eventually (with fully automagic operations on virtual machines that the end user does not actually use directly)

If retroshare does not work out, there are many other ways to do file sharing (perhaps with less privacy/anonymity features that complicate installation) eg here's the Chocolatey list:

https://community.chocolatey.org/packages?q=tag%3Afilesharing

If anyone recommends one of those, please comment here. Will need to check for compatible file share apps on both Linux and MacOS (likely though perhaps with different names).

Another approach that will be viable long term is using IPFS. 

[Brave browser](https://brave.com/download/) is worth having on all 3 platforms and can download directly (via a web gateway) from an IPFS link sent by email in much the same way as downloading from Library Genesis or Anna's Archive (which store their stuff on IPFS). Can lightly encrypt the file in a compressed archive for additional privacy.

cURL command line utility v8.4.0 released 2023-10-11 also retrieves files from IPFS the same way and is available for MacOS via HomeBrew and for Windows via Chocolatey.

I could probably work out how to upload to IFPS and email suitable links without wasting too much time that I would not have to do eventually anyway. So David could just pass the USB to me and everybody install Brave and/or cURL.

## Conclusion

1. Windows users should install Chocolatey now. (MacOS users will use HomeBrew with "brew --cask").
2. All should  install  Brave browser now.
3. I may report back that people should use Brave browser (or latest version of cURL) with links I will email,  if I don't report back that all should install retroshare (with configuration details that I will provide).
4. It would be good to also have a much smaller word processor file from extracting the OCR from the .pdf.
(But we would still need to pass around the large .pdf)
